.github/workflows/main.yml

```diff
@@ -0,0 +1,28 @@
+name: CI
+
+on:
+  push:
+    branches: [ master ]
+  pull_request:
+    branches: [ master ]
+
+  workflow_dispatch:
+
+jobs:
+  build:
+    runs-on: macos-latest
+
+    steps:
+      - uses: actions/checkout@v2
+      
+      - name: update bundler
```

Comment:
@Zandor300 you might check out https://github.com/ruby/setup-ruby instead of manually doing these various Ruby setup steps.

Reply:
Nice! Thanks!